oshc/main/templates/registration/registration_form.html

```diff
@@ -21,11 +21,16 @@ <h1 class="form-title text-center">{% trans 'Sign Up Here' %}</h1>
               </p>
               <p class="required"><label class="required" for="id_password1">
                                     {% trans 'Password:' %}</label>
-                <input class="form-control" id="id_password1" name="password1" type="password" required/>
-              </p>
+                <input class="form-control" id="id_password1" name="password1"  type="password" required/><br>
```

Comment:
You use IDs for JavaScript manipulation.
You use Names for form field submission.
The two are not related. So setting both to the same value is OK